assets/src/design-system/components/banner/index.js

```diff
@@ -27,9 +27,9 @@ import styled, { css } from 'styled-components';
 import { Close } from '../../icons';
 import { THEME_CONSTANTS } from '../../theme';
 import { Button, BUTTON_SIZES, BUTTON_TYPES, BUTTON_VARIANTS } from '../button';
-import { Text } from '../typography';
+import { Headline } from '../typography';
 
-const Title = styled(Text)`
+const Title = styled(Headline)`
```

Comment:
Semantically I think the title should be a heading tag. Do y'all think we should leave it as a `p` tag?

Reply:
great update! thank you!